README.md

```diff
@@ -148,7 +150,248 @@ wind.add(textfield);
+
+### Feature vs Platform
+[Feature vs Platform]: #feature-vs-platform
+
+When do you use [Feature] detection instead of just changing the logic based on the current [Platform]? Using feature detection allows you minimize the concerns of your logic, allowing it to be a single unit that does not rely on anything else unrelated.
+
+Consider the following [Platform] detection logic:
+
+````js
+var UI = require('ui');
+var Platform = require('platform');
+
+var isAplite = (Platform.version() === 'aplite');
+var isChalk = (Platform.version() === 'chalk');
+var card = new UI.Card({
+  title: 'Example',
+  titleColor: isAplite ? 'black' : 'dark-green',
+  subtitle: isChalk ? 'Hello World!' : 'Hello!',
+  body: isAplite ? 'Press up or down' : 'Speak to me',
+});
+
+card.show();
+````
+
+The first issue has to do with future proofing. It is checking if the current Pebble has a round screen by seeing if it is on Chalk, however there may be future platforms that have round screens. It can instead use [Feature.round()] which will update to include newer platforms as they are introduced.
+
+The second issue is unintentional entanglement of different concerns. `isAplite` is being used to both determine whether the Pebble is black and white and whether there is a microphone. It is harmless in this small example,  but when the code grows, it could potentially change such that a function both sets up the color and interaction based on a single boolean `isAplite`. This mixes color presentation logic with interaction logic.
+
+Consider the same example using [Feature] detection instead:
+
+````js
+var UI = require('ui');
+var Feature = require('platform/feature');
+
+var card = new UI.Card({
+  title: 'Example',
+  titleColor: Feature.color('dark-green', 'black'),
+  subtitle: Feature.round( 'Hello World!', 'Hello!'),
+  body: Feature.microphone('Speak to me', 'Press up or down'),
+});
+
+card.show();
+````
+
+Now, if it is necessary to separate the different logic in setting up the card, the individual units can be implemented in separate functions without anything unintentionally mixing the logic together. [Feature] is provided as a module, so it is always available where you decide to move your logic.
+
+The two examples consist of logic confined into one line, but if each line was instead large blocks of logic with the `isAplite` boolean used throughout, the issue is more apparent, hence the recommendation to use [Feature] detection. Of course, for capabilities or characteristics that [Feature] is unable to allow you to discern, use [Platform].
+
+**NOTE:** [Feature] underneath the hood does not actually perform feature detection. The benefit of using [Feature] however is that the detection logic is abstracted out of your application, so if any platform is changed or introduced, Pebble.js will update and your application will automatically adapt in the case that no new capabilities or characteristics are introduced.
```

Comment:
"[Feature] underneath the hood does not actually perform feature detection. "

What does this mean, and why do I care about it as a developer?